backend/api.go

```diff
@@ -0,0 +1,126 @@
+package backend
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"path"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/stellar/project-viewer/internal/queries"
+	"google.golang.org/api/option"
+)
+
+const projectID = "test-project-291320"
+const keyFileName = "testingKey.json"
+
+var client *bigquery.Client
+
+func getBigQueryClient() (*bigquery.Client, error) {
```

Comment:
I think the `api.go` file should only include the definitions of actual functions used in the API (i.e., implementations of user-facing endpoints). Mind moving this function into a separate file, like `bigquery_client.go`? Same package is totally fine.